src/components/form/form.types.ts

```diff
@@ -282,3 +311,9 @@ export enum FormLayoutType {
      */
     Row = 'row',
 }
+
+/**
+ * @public
+ * Represents the JSON schema with Lime specific options
+ */
+export interface LimeFormSchema extends JSONSchema7 {}
```

Comment:
Any reason why this is prefixed with `Lime`? Most other public types/interfaces are not :thinking:

Reply:
No, we can remove it. `FormSchema` maybe is better. I will update it

Reply:
Fixed!